.github/workflows/code-samples.yml

```diff
@@ -7,7 +7,7 @@ on:
     paths: [ 'code_samples/**','.github/workflows/code-samples.yml' ]
 
 env:
-  PLUGIN_VERIFIER_IDE_VERSIONS: '2022.3.3 2023.1.5 2023.2.3'
+  PLUGIN_VERIFIER_IDE_VERSIONS: '2023.1.5 2023.2.5'
```

Comment:
I assume it will be 2023.3 after release. Leaving the comment to not forget.
Is it possible to set 2023.3 Beta here?

Reply:
yes 2023.3 GA will be added then